Gems/LmbrCentral/Code/Source/Shape/AxisAlignedBoxShape.h

```diff
@@ -40,5 +40,8 @@ namespace LmbrCentral
 
         // AZ::TransformNotificationBus::Handler
         void OnTransformChanged(const AZ::Transform& local, const AZ::Transform& world) override;
+
+        // BoxShapeComponentRequestBus::Handler overrides ...
+        bool IsTypeAxisAligned() override;
```

Comment:
suggestion: Is it worth making this `const` (I know it's a bus call but nowadays it's legal to do this and arguably still useful - if people do want to cache something they can always fallback to `mutable`)